### PR TITLE
Ghactions

### PR DESCRIPTION
**Pull request by sjsrey**

### CODE REVIEW

File: .github/workflows/pythonapp.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.y
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
```
Comment (knaaptime) [inline .github/workflows/pythonapp.yml:25]: any idea if it's possible to use conda here? pip chokes on some of the deps (e.g. geopandas)

Comment (sjsrey) [reply .github/workflows/pythonapp.yml:25]: That was just to get the stub setup. I'm revising for conda now.

**Comment by knaaptime** [comment]
image

that may have something to do with it. I've also tried installing the certs manually but there's a unix path error (I think because its calling the script from bash on windows). So then i went down a different rabbit hole trying to run the windows tests separately using CMD instead, but now hitting a different path problem that it seems to be using the wrong python. 

All told, I think this is just a rough edge with using conda on windows for gh-actions and should get resolved elsewhere before too long. Im happy adopting this new testing scheme... maybe there's a way we can allow windows failures until we can get this smoothed over?

**Comment by sjsrey** [comment]
Passing on mac and linux. I think we should merge and add a different pr to focus on debugging windows.

**Pull request merged**